qutebrowser/browser/adblock.py

```diff
@@ -174,15 +174,13 @@ def adblock_update(self):
         for url in config.val.content.host_blocking.lists:
             if url.scheme() == 'file':
                 filename = url.toLocalFile()
-                try:
-                    fileobj = open(filename, 'rb')
-                except OSError as e:
-                    message.error("adblock: Error while reading {}: {}".format(
-                        filename, e.strerror))
-                    continue
-                download = FakeDownload(fileobj)
-                self._in_progress.append(download)
-                self.on_download_finished(download)
+                if os.path.isdir(filename):
+                    for dirpath, _, filenames in os.walk(filename):
```

Comment:
Maybe this should use `os.scandir` instead? ([docs](https://docs.python.org/3.5/library/os.html#os.scandir))

Reply:
I used ```os.walk()``` in order to easily import all files below the specified directory.
Would you like to keep this functionality and just use ```os.scandir()``` instead or should this only import files from the specified dir and no subdirs?

Reply:
Ah, good point. I personally don't think it should recurse subdirectories, I find that rather unexpected.

Reply:
done